src/renderer/global.scss

```diff
@@ -129,12 +130,12 @@ body {
     width: 8px;
 }
 
-[data-theme='dark'] ::-webkit-scrollbar-thumb {
+[data-theme="dark"] ::-webkit-scrollbar-thumb {
```

Comment:
I'm pretty sure we use single-quotes everywhere, so why? Did your default formatter do that? If so, then maybe we should add stylelint. Since we plan on moving away from CSS-in-JS, adding stylelint might be a good move regardless.

Reply:
Different pr please, but yes. I didn't do this on purpose so it has to have been formatter